kubetest/e2e.go

```diff
@@ -506,16 +518,21 @@ func KubemarkTest(dump string, o options) error {
 	})
 
 	// Start new run
-	if popN, err := pushEnv("NUM_NODES", o.kubemarkNodes); err != nil {
+	if popN, err := pushEnv("NUM_NODES", nodes); err != nil {
 		return err
 	} else {
 		defer popN()
 	}
-	if popM, err := pushEnv("MASTER_SIZE", o.kubemarkMasterSize); err != nil {
+	if popM, err := pushEnv("MASTER_SIZE", masterSize); err != nil {
```

Comment:
Aka if you strip these flags from your jobs so they auto-calculate and remove the flag I won't complain (at least until I finish refactoring what happens before kubetest and focus on eliminating bash scripts between kubetest and actual tests running, but that is much further down the line)